content/tutorials/basic-movement.md

```diff
@@ -2,264 +2,164 @@
 title: Basic Movement
 ---
 
-The aim of this tutorial is to get a motor turning with your kit.
-To complete this tutorial, you'll need the following:
+The aim of this tutorial is to get a motor turning with your kit. To complete this tutorial, you'll need the following:
 
-* The [power board](/docs/kit/power_board)
+* The [power board](/kit/power-board)
+* A [motor board](/kit/motor-board)
+* The [Pi](/kit/pi)
 * The battery (charged, of course)
-* The cable to connect the battery to the power board
-* A [motor board](/docs/kit/motor_board)
 * 2 large (7.5mm) green _CamCon_ connectors to plug the power board and motor board together
 * 2 lengths of a suitable gauge of wire for powering the motor board from the power board (one should be black/grey)
 * A motor (see specification below)
 * A small (5mm) green _CamCon_ connector to plug the motor and motor board together
 * 2 lengths of a suitable gauge of wire for your motor
-* A Micro USB cable
-* A USB hub
-* A standard USB cable
+* 2 Micro USB cables
+* The Pi power cable
+
+You'll also need these, which aren't provided with your kit:
+
 * The memory stick
 * A soldering Iron
 * Some solder wire
 * Wire strippers
 * A small slotted/flat blade screwdriver (for the _CamCon_ screws)
 
-You should be familiar with the setup for most of the above now, so it's just
-the motor-related parts that need explaining.
-
+You should be familiar with the setup for most of the above now from the [kit assembly tutorial](/tutorials/kit-assembly/), so it's just the motor-related parts that need explaining.
 
-Motor Specification
--------------------
-
-There is a certain specification the motor(s) you use must meet.
-The criteria are as follows:
+## Motor Specification
+There is a certain specification the motor(s) you use must meet. The criteria are as follows:
 
 * 12V motor
-* A stall current of less than 10A (this is the current limit for the [motor boards](/docs/kit/motor_board))
-
-<div class="info">
-    When designing your robot you should bear in mind that while each motor board can deliver 10A on each output,
-    all the power needs to go through the power board, which is fused at 15A.
-    This means that across all the outputs for all the motors,
-    you can only draw up to 15A at any time.
-</div>
+* A stall current of less than 10A (this is the current limit for the [motor boards](/kit/motor_board)
 
-Connecting a Motor
-------------------
+{{% notice note %}}
+When designing your robot you should bear in mind that while each motor board can deliver 10A on each output, all the power needs to go through the power board, which is fused at 15A. This means that across all the outputs for all the motors, you can only draw up to 15A at any time.
```

Comment:
I'm not sure where the 15A figure came from, but now would be a good time to correct it - the 'H' outputs on the power board are software-limited to 20A, and the whole kit is software-limited to 30A. There's also a 40A hard fuse in case the software limiting fails. Off the top of my head there's nowhere that a 15A limit is implemented.

See upstream: https://studentrobotics.org/docs/kit/power_board#Specification